packages/filezilla.rb

```diff
@@ -44,11 +44,10 @@ def self.build
 
   def self.install
     system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
-    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/env.d/"
-    filezilla = <<~'EOF'
+    filezilla = <<~EOF
```

Comment:
`File.write 'filezilla_envd', <<~EOF` and `FileUtils.install 'filezilla_envd', "#{CREW_DEST_PREFIX}/etc/env.d/10-filezilla", mode: 0o644` should fix this...